src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -219,7 +224,24 @@ public async Task RunAsync()
             }
         }
 
-        public string GetNodeLocation()
+        private async Task<bool> CheckIfNode20ResultsInGlibCError()
+        {
+            bool node20ResultsInGlibCError = false;
+
+            bool UseNode20InUnsupportedSystem = AgentKnobs.UseNode20InUnsupportedSystem.GetValue(ExecutionContext).AsBoolean();
+
+            if (!UseNode20InUnsupportedSystem)
+            {
+                var node20 = Path.Combine(HostContext.GetDirectory(WellKnownDirectory.Externals), NodeHandler.Node20_1Folder, "bin", $"node{IOUtil.ExeExtension}");
+                string node20TestCmd = $"bash -c \"{node20} -v\"";
+                List<string> nodeVersionOutput = await ExecuteCommandAsync(ExecutionContext, node20, "-v");
```

Comment:
Is `node20TestCmd `being used?  `List<string> nodeVersionOutput = await ExecuteCommandAsync(ExecutionContext, node20TestCmd, "-v");`